redis/service.cc

```diff
@@ -38,12 +38,12 @@ redis_service::~redis_service()
 {
 }
 
-future<> redis_service::listen(distributed<auth::service>& auth_service, db::config& cfg)
+future<> redis_service::listen(seastar::sharded<auth::service>& auth_service, db::config& cfg)
```

Comment:
I still don't understand why you like to put "seastar::" specifically on sharded, but I don't mind it either, so I'll commit.

Reply:
It's a good coding convention to use fully-qualified names, but, of course, if `scylla.git` has different preferences, I'll adjust the series for that. Should I replace the `using namespace seastar` with `#include "seastarx.hh"`?

Reply:
It is historical. `sharded` was the first type in the seastar namespace (scylladb/seastar@ab7be2890feeb689288b08b0961a86a5ef901436), so we started using it in its fully qualified form.

Reply:
I agree it's good form to use a namespace qualifer. We can decide to accept both forms but prefer the fully qualified form for the next few years.